src/instance/InstanceController.ts

```diff
@@ -59,7 +59,7 @@ export class InstanceController {
   @Delete(':instanceId')
   @ApiOperation({summary: 'Delete an instance.'})
   @ApiResponse({description: 'The instance has successfully deleted.', status: 200})
-  @ApiResponse({description: 'Instance not found', status: 404})
+  @ApiResponse(status404instance)
```

Comment:
Name it `status404description` to be consistent with `status422description` and `status500description`.

Reply:
That's instance specific, while the other ones are generic HTTP errors